frontend/src/components/base/Button/Button.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { fireEvent, render } from '../../../test-utils';
+import { Button } from './Button';
+
+describe('Button', () => {
+  it('should render without errors', () => {
```

Comment:
This should also be fine; no need to change just FYI

```suggestion
  it('should render', () => {
```

Reply:
Short and concise! Done.